gnovm/stdlibs/std/native.gno

```diff
@@ -14,41 +14,41 @@ func GetChainID() string     // injected
 func GetChainDomain() string // injected
 func GetHeight() int64       // injected
 
-func GetOrigSend() Coins {
-	den, amt := origSend()
+func OriginSend() Coins {
+	den, amt := originSend()
 	coins := make(Coins, len(den))
 	for i := range coins {
 		coins[i] = Coin{Denom: den[i], Amount: amt[i]}
 	}
 	return coins
 }
 
-func GetOrigCaller() Address {
-	return Address(origCaller())
+func OriginCaller() Address {
+	return Address(originCaller())
 }
 
 func CurrentRealm() Realm {
 	addr, path := getRealm(0)
 	return Realm{Address(addr), path}
 }
 
-func PrevRealm() Realm {
+func PreviousRealm() Realm {
 	addr, path := getRealm(1)
 	return Realm{Address(addr), path}
 }
 
-func GetOrigPkgAddr() Address {
-	return Address(origPkgAddr())
+func GetOriginPkgAddress() Address {
```

Comment:
Can you remove it here?

Reply:
I dont see `func GetOrigPkgAddr() Address {`, it has been already refactored to `func GetOriginPkgAddress() Address {`

Reply:
Refactored in https://github.com/gnolang/gno/pull/3374/commits/5c6abafb639ee30bab242b8af27dcd110f014209